package.json

```diff
@@ -12,12 +12,14 @@
     "clean": "rm -rf ./packages/*/lib ./packages/*/*.tsbuildinfo",
     "clean:nm": "rm -rf ./packages/*/node_modules ./node_modules",
     "build": "lerna run build",
+    "build:bundle": "lerna run build:bundle",
     "build:watch": "lerna exec --parallel -- 'yarn run build:watch'",
     "build:ifchanged": "lerna exec -- ../../scripts/build_if_changed.sh",
-    "lint": "eslint --report-unused-disable-directives --color --ext .ts packages/*/src packages/*/test",
+    "lint": "eslint --report-unused-disable-directives --color --ext .ts packages/*/src packages/*/test --ignore-pattern webEsmBundle.browser.test.ts",
```

Comment:
wouldn't it be better to have the ignored files as part of eslint config? see https://eslint.org/docs/latest/use/configure/ignore#ignoring-files